pkgs/servers/monitoring/nagios-plugins/check_interfaces/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  check_interfaces,
+  fetchurl,
+  lib,
+  net-snmp,
+  nix-update-script,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "check_interfaces";
+  version = "1.4.3";
+
+  src = fetchurl {
+    url = "https://github.com/NETWAYS/check_interfaces/releases/download/v${version}/check_interfaces-${version}.tar.gz";
+    hash = "sha256-ePJ+T9/le0oz3dRtueWZuZKzBtg1n1Y+Z2Rgh1YBfXs=";
+  };
+
+  buildInputs = [ net-snmp ];
+
+  configureFlags = [ "--libexecdir=${placeholder "out"}/bin" ];
+
+  enableParallelBuilding = true;
+
+  postInstall = ''
+    # Remove unnecessary header files
+    rm --recursive $out/include
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
```

Comment:
The testVersion test does not work and is failing with:

> check_interfaces --version returned a non-zero exit code.

Reply:
I found a workaround, looks like using the full path to the check_interfaces binary is a resolution:

```nix
  command = "${check_interfaces}/bin/check_interfaces --version";
```

Maybe this is related to this check in the code: https://github.com/NETWAYS/check_interfaces/blob/v1.4.3/check_interfaces.c#L1295?

Reply:
I guess nobody ever put it into `$PATH` :-)
Anyway, fixed in v1.4.4.

Reply:
Thanks @RincewindsHat! I've updated to v1.4.4.